mesonbuild/modules/wayland.py

```diff
@@ -112,11 +112,13 @@ def scan_xml(self, state: ModuleState, args: T.Tuple[T.List[FileOrString]], kwar
         'wayland.find_protocol',
         KwargInfo('state', str, default='stable', validator=in_set_validator({'stable', 'staging', 'unstable'})),
         KwargInfo('version', (int, NoneType)),
+        KwargInfo('vendored', bool, default=False, since='1.6.0'),
```

Comment:
How can I indicate that this should be the next release?

Reply:
Just write the next release like you have, if this doesn't make it in till the next release, then we'd review saying "please update this to the next release"